frontend/src/components/Utils/time.js

```diff
@@ -78,4 +76,32 @@ export function timezonesForCountry(countryName) {
   return zones.map(e => {
     return e.replace(/[_]/g, ' ');
   });
+}
+
+/**
+ * Get a date in 'YYYY-MM-DD' form. 
+ * 
+ * @param {string} msTimestamp Timestamp, in milliseconds since epoch.
```

Comment:
Based on what I saw in PR #75, `activity[DB.ACTIVITIES_START_TIME]` was of type long. If that is the case, why would `msTimestamp` be of type string rather than int (also I think for JS, there is no such type as `int`, only `number`)?

Reply:
it's a number.